code/modules/boh/culture_descriptor/faction/factions_sergal.dm

```diff
@@ -16,6 +16,7 @@
 	Since the year of the Rain-Calendar 20, it was unified as the name of the Kingdom and Rain Silves' clan, formed from a conglomerate of unified tribes under one flag.\
 	The 'Shigu Kingdom' collapsed during the late stage of world war. And after the end of the war, it reconstructed as the 'Shigu Dukedom', developed as a large city of Northern Tal."
 
+
```

Comment:
```suggestion

```

Reply:
Random spaces bad

Reply:
yup.
I accidentally messed with this part of the code